### PR TITLE
Enhanced UI responsiveness

### LINKED ISSUE #2042: Enhance UI responsiveness
Part of https://github.com/bigbinary/neeto-engineering-ui-ux/issues/221

Todo

- [x] Alert
- [x] Modal
- [x] Pane
- [x] Toastr
- [x] NoData
- [x] DatePicker
- [x] Stepper

@goutham-subramanyam _a

### PR DESCRIPTION
**Pull request by goutham-subramanyam**
- Fixes #2042

**Description**
Enhanced the responsiveness of the following components

- Alert
- Modal
- Pane
- Toastr
- NoData
- DatePicker
- Stepper

**Screenshots**

Screenshot 2023-12-12 at 3 22 21 AM
Screenshot 2023-12-12 at 3 23 48 AM
Screenshot 2023-12-12 at 3 26 02 AM
Screenshot 2023-12-12 at 3 26 22 AM

**Checklist**

- [ ] ~I have made corresponding changes to the documentation.~
- [ ] ~I have updated the types definition of modified exports.~
- [ ] ~I have verified the functionality in some of the neeto web-apps.~
- [ ] ~I have added tests that prove my fix is effective or that my feature works~
- [ ] ~I have added the necessary label (patch/minor/major - If package publish
      is required).~

**Reviewers**
@praveen-murali-ind _a Please review.

### CODE REVIEW

File: src/styles/components/_pane.scss [hunk 1]
```diff
@@ -60,6 +60,9 @@
     transform: translateX(100%);
     transition: all 0.3s;
 
+    display: flex;
```
Comment (praveen-murali-ind) [inline src/styles/components/_pane.scss:63]: @goutham-subramanyam _a Let’s revert these changes for Pane as it has some side effects.

File: src/styles/components/_pane.scss [hunk 2]
```diff
@@ -94,6 +97,7 @@
     padding: var(--neeto-ui-pane-body-padding-y) var(--neeto-ui-pane-body-padding-x);
     font-size: var(--neeto-ui-pane-body-font-size);
     overflow-y: auto;
+    flex-grow: 1;
```
Comment (praveen-murali-ind) [inline src/styles/components/_pane.scss:100]: @goutham-subramanyam _a Revert the change to the Pane layout for now, until we confirm it doesn't affect existing products.

**Comment by goutham-subramanyam** [comment]
@praveen-murali-ind _a Reverted the changes. Please check.

**Review by praveen-murali-ind — APPROVED** [review]
LGTM.

**Review by ajmaln — APPROVED** [review]
